UI/GroceyStore_UI/grocery_store/src/History.js

```diff
@@ -1,17 +1,113 @@
+import TableHead from '@material-ui/core/TableHead';
+import TablePagination from '@material-ui/core/TablePagination';
+import TableRow from '@material-ui/core/TableRow';
 
-class History extends Component {
+const columns = [
+    { id: 'date', label: 'Date', minWidth: 130 },
+    { id: 'time', label: 'Time', minWidth: 130 },
+    {
+      id: 'status',
+      label: 'Status',
+      minWidth: 130,
+      //align: 'right',
+      //format: (value) => value.toLocaleString('en-US'),
+    },
+  ];
 
-    render() {
-        return (
-            <div>
-                <Header title={(this.props.location.pathname).substring(1)} />
-                This is History page
-            </div>
-        )
-    }
+  function createData(date, time, status) {
+    return { date, time, status };
+  }
 
-}
+  const rows = [
+    createData('07-07-2020', '1:00 PM', 'Delivered'),
+    createData('09-07-2020', '2:00 PM', 'Our for Delivery'),
+    createData('11-07-2020', '3:00 PM', 'Delivered'),
+    createData('13-07-2020', '4:00 PM', 'Delivered'),
+    createData('15-07-2020', '5:00 PM', 'Delivered'),
+    createData('17-07-2020', '6:00 PM', 'Our for Delivery'),
+    createData('19-07-2020', '7:00 PM', 'Delivered'),
+    createData('21-07-2020', '8:00 PM', 'Delivered'),
+    createData('22-07-2020', '9:00 PM', 'Delivered'),
+  
+  ];
+  
+  const useStyles = makeStyles({
+    root: {
+      width: '100%',
+    },
+    container: {
+      maxHeight: 440,
+    },
+  });  
 
-export default History;
+  export default function History() {
+    const classes = useStyles();
+    const [page, setPage] = React.useState(0);
+    const [rowsPerPage, setRowsPerPage] = React.useState(10);
+  
+    const handleChangePage = (event, newPage) => {
+      setPage(newPage);
+    };
+  
+    const handleChangeRowsPerPage = (event) => {
+      setRowsPerPage(+event.target.value);
+      setPage(0);
+    };
+  
+    return (
+      <Paper className={classes.root}>
+        <TableContainer className={classes.container}>
+          <Table stickyHeader aria-label="sticky table">
+            <TableHead>
+              <TableRow>
+                {columns.map((column) => (
+                  <TableCell
+                    key={column.id}
+                    align={column.align}
+                    style={{ minWidth: column.minWidth }}
+                  >
+                    {column.label}
+                  </TableCell>
+                ))}
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {rows.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row) => {
+                return (
+                  <TableRow hover role="checkbox" tabIndex={-1} key={row.code}>
+                    {columns.map((column) => {
+                      const value = row[column.id];
+                      return (
+                        <TableCell 
+                        numeric component="a" href="/Notification"

```

Comment:
Why are we navigating to 'Notification' directly from history? Am i missing something? Even if we need to navigate, use react router

Reply:
I used it for testing purpose.